packages/odyssey-react/src/components/Table/TableHeaderCell.tsx

```diff
@@ -46,4 +49,11 @@ const TableHeaderCell = forwardRef<Ref, Props>((props, ref) => {
   );
```

Comment:
similar feedback here and for all the other statics about the order of our spread `{...omitProps}` we want these to be first (and really need to write a fixable lint rule for it :soon:)